python/ga4gh/dos/client.py

```diff
@@ -0,0 +1,31 @@
+# Simple client usage via bravo
+
+import os
+
+from bravado.client import SwaggerClient
+# from bravado.swagger_model import load_file
+
+SWAGGER_FILENAME = 'data_objects_service.swagger.json'
+current_directory = os.path.dirname(os.path.realpath(__file__))
+SWAGGER_PATH = os.path.join(current_directory, SWAGGER_FILENAME)
```

Comment:
When the user installs the package, will the swagger file be included?   It would be useful.

Reply:
Given a dict:
```
{'updated': '2017-10-23T16:20:45+00:00', 'created': '2017-10-23T16:20:45+00:00', 'urls': [{'url': 's3://test/my-bucket/passport+photo.JPG', 'system_metadata': {'bucket_name': 'my-bucket', 'StorageClass': 'STANDARD', 'event_type': 'ObjectCreated:Put'}, 'user_metadata': {'foo': 'bar'}}], 'file_size': 341005, 'checksums': [{'checksum': 'd3b3a66c7235c6b09a55a626861f5f91', 'type': 'md5'}], 'id': 'passport+photo.JPG'}
```
After unmarshal(), there seems to be an extra 'fields' attribute in metadata
```
ga4ghDataObject(aliases=None, checksums=[{'checksum': 'd3b3a66c7235c6b09a55a626861f5f91', 'type': 'md5'}], created=datetime.datetime(2017, 10, 23, 16, 20, 45, tzinfo=tzutc()), description=None, id='passport+photo.JPG', mime_type=None, name=None, size=None, updated=datetime.datetime(2017, 10, 23, 16, 20, 45, tzinfo=tzutc()), urls=[{'url': 's3://test/my-bucket/passport+photo.JPG', 'system_metadata': {u'fields': None, 'bucket_name': 'my-bucket', 'StorageClass': 'STANDARD', 'event_type': 'ObjectCreated:Put'}, 'user_metadata': {u'fields': None, 'foo': 'bar'}}], version=None)
```

Reply:
Yes, the swagger is currently included https://github.com/ga4gh/data-object-schemas/pull/24/files#diff-2eeaed663bd0d25b7e608891384b7298R6 , setting this up to work with release/packaging will be forthcoming.